third_party/googletest/src/googletest/include/gtest/gtest.h

```diff
@@ -1349,9 +1349,21 @@ template <typename T1, typename T2>
 AssertionResult CmpHelperEQ(const char* lhs_expression,
                             const char* rhs_expression, const T1& lhs,
                             const T2& rhs) {
+#if defined(_MSC_VER)
```

Comment:
A lot of ugly here to silence a warning on a single line. Annoyingly this is in a header too, so moving it into GN code would propagate those snoozed warnings all over the place. I don't know if we have a better solution but eventually this should get cleaned up somehow.

Reply:
b/288165638